<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>凭证搜索</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbCondition">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>时间范围</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="rdoAll">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>所有账</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="rdoCurSuilte">
             <property name="text">
              <string>当前帐套</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rdoCurPzSet">
             <property name="text">
              <string>当前凭证集</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="rdoDateLimit">
             <property name="text">
              <string>指定时间范围</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>开始时间</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="deStart">
             <property name="displayFormat">
              <string>yyyy-M</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>结束时间</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="deEnd">
             <property name="displayFormat">
              <string>yyyy-M</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="gbSummary">
            <property name="title">
             <string>摘要中包含</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QRadioButton" name="rdoInvoiceNum">
               <property name="text">
                <string>发票号</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="edtInvoice"/>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="rdoAllContent">
               <property name="text">
                <string>任意内容</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="edtContent"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbSub">
            <property name="title">
             <string>科目</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>一级科目</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="SubjectSelectorComboBox" name="cmbFSub"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>二级科目</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SubjectSelectorComboBox" name="cmbSSub"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="gbMoney">
          <property name="title">
           <string>金额</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>金额范围</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="chkPreciseMatch">
             <property name="text">
              <string>精确匹配</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblMax">
             <property name="text">
              <string>最大</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="spnMaxValue">
             <property name="minimum">
              <double>-100000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblMin">
             <property name="text">
              <string>最小</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="spnMinValue">
             <property name="minimum">
              <double>-100000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>借贷方向</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="chkJDir">
             <property name="text">
              <string>借方</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="chkDDir">
             <property name="text">
              <string>贷方 </string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0">
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPageNum">
       <property name="text">
        <string>state</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPrePage">
       <property name="toolTip">
        <string>上一页</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/go_previous.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnNextPage">
       <property name="toolTip">
        <string>下一页</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/go_next.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnExpand">
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tw">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>凭证号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>摘要</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>主目</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>子目</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>币种</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借方</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>贷方</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SubjectSelectorComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/subjectselectorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
